conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -64,6 +64,6 @@ build: off
 test_script:
     - "%CMD_IN_ENV% conda build {{ recipe_dir }} --quiet"
 deploy_script:
-{% for owner, channel in channels['targets'] %}
-    - 'python ci_support\upload_or_check_non_existence.py .\{{ recipe_dir }} {{ owner }} --channel={{ channel }}'
+{%- for owner, channel in channels['targets'] %}
+    - cmd: upload_or_check_non_existence.py .\{{ recipe_dir }} {{ owner }} --channel={{ channel }}
```

Comment:
Have tried a few variations of this and Windows likes none of them. Maybe we can add a batch script that calls the Python file IDK. Going to let someone else resolve that and we can try this again.